.circleci/config.yml

```diff
@@ -4,9 +4,21 @@ orbs:
   win: circleci/windows@2.4.0
 
 executors:
-  circleci_machine:
+  circleci_machine_37:
     docker:
-      - image: cimg/node:14.15
```

Comment:
any reason why we are taking away the image for `node:14.15`?

Reply:
so for linux, we are no longer using conda or python virtual environments. we are just using docker -- and on docker we can have only one image. hence we have a python image with latest node. so if you see my dev notes, currently on circleCI we are running node 16 but surprisingly all the javascript tests are running properly and passing (when they should be failing) 

Anyhow this is a mystery! but for now since the test are actually being processed and running as expected instead of being skipped we have left it as is.